lib/hdiutil.js

```diff
@@ -2,10 +2,10 @@ var fs = require('fs')
 var temp = require('fs-temp')
 var util = require('./util')
 
-exports.convert = function (source, target, cb) {
+exports.convert = function (source, format, target, cb) {
   var args = []
   args.push('convert', source)
-  args.push('-format', 'UDZO')
+  args.push('-format', format)
   args.push('-imagekey', 'zlib-level=9')
```

Comment:
Actually, `NONE` is not a valid value at all. The following, and some more mostly deprecated, are valid values:

```
[
  "UDRW",
  "UDRO",
  "UDCO",
  "UDZO",
  "ULFO",
  "UDBZ"
]
```

I changed quite a lot of the code right after merging. The tests didn't pass and the functionality was broken.

The default value is, as siad, set in other parts of appdmg. I prefer to keep it as a mandatory option to the hdiutil functions for now...